web-js/4.x/client-side-feature-layer/README.MD

```diff
@@ -0,0 +1,6 @@
+# Client Side Feature Layer
+
+This sample shows how create a client side Feature Layer so that you can use labels.
```

Comment:
Could you please add a little more detail about how this sample works? You can reference some of the other 3.x/4.x samples for examples.